doc/devguide/extending/app-layer/transactions.rst

```diff
@@ -0,0 +1,310 @@
+************
+Transactions
+************
+
+.. contents:: Table of Contents
+
+General Concepts
+================
+
+For Suricata, transactions are abstractions that help with detecting and logging. An example of complete transaction is
+a pair of messages Request (from client to server) and Response (from server to client) in HTTP.
```

Comment:
a pair of messages in the form of a request (from client to server) and a response (from server to client) in HTTP.